backend/pkg/tools/openapi-generator/main.go

```diff
@@ -481,6 +482,82 @@ func updateUsers(doc3 *openapi3.T) {
 	}
 }
 
+func updateTransforms(doc3 *openapi3.T) {
```

Comment:
I assume we need to add an entirely new node to document the deploy trnasforms endpoint?

Reply:
wasn't sure how to handle that. i had initially used the same semantics as the rest but because this endpoint is not grpc it can't be done that way. do you want me to just put a custom explanation there and call it good? or something else in mind?

Reply:
We need to add the node for the WASM endpoint there so that the result in the OpenAPI spec is similar to what is generated for the other endpoints. Would need to do a bit research / check the library a bit to figure out what to do exactly as well.